### PR TITLE
tests/converter: modify unpack testcase

### PR DESCRIPTION
**Pull request by SToPire**
This patch modifies the unpack testcase to keep consistent with the new `nydus-image unpack` interface in nydus-image.

### CODE REVIEW

**Comment by SToPire** [comment]
The smoke test should work properly if [this](https://github.com/dragonflyoss/nydus/pull/1606) PR is applied.

File: pkg/converter/tool/builder.go [hunk 1]
```diff
@@ -305,7 +305,41 @@ func Unpack(option UnpackOption) error {
 	}
 
 	if option.BackendConfigPath != "" {
-		args = append(args, "--backend-config", option.BackendConfigPath)
+		configFile, err := os.Open(option.BackendConfigPath)
```
Comment (imeoer) [inline pkg/converter/tool/builder.go:308]: It seems the `configFile` not used.

File: pkg/converter/tool/builder.go [hunk 2]
```diff
@@ -305,7 +305,41 @@ func Unpack(option UnpackOption) error {
...
+		if !ok {
+			return errors.New("failed to get backend config with type " + backendConfigType.(string))
+		}
+
+		backendConfigBytes, err := json.Marshal(backendConfig)
+		if err != nil {
+			return errors.Wrapf(err, "fail to marshal backend config %v", backendConfig)
+		}
+
+		logrus.Errorf("Backend config: %s", backendConfigBytes)
```
Comment (imeoer) [inline pkg/converter/tool/builder.go:339]: The backend config maybe contains secret data, we shouldn't print it.

**Review by imeoer — COMMENTED** [review]
Thanks! others LGTM, should we create a new builder (nydus-image) release?

**Pull request merged**